data/avgi/avgi ships.txt

```diff
@@ -0,0 +1,2142 @@
+		licenses
+			"Avgi Military"
+		"shields" 4200
+		"hull" 27600
+		"required crew" 60
+		"bunks" 90
+		"mass" 450
+		"drag" 3.6
+		"heat dissipation" 1.8
+		"fuel capacity" 2400
+		"cargo space" 90
+		"outfit space" 690
+		"weapon capacity" 180
+		"engine capacity" 174
+		"anchor point" 3
+		weapon
+			"blast radius" 540
+			"shield damage" 6400
+			"hull damage" 4800
+			"hit force" 4200
+	outfits
+		"Fleck Turret"
+		"Fleck Round" 2400
+		"Phased Array Laser (Blue)" 2
+		"Phased Array Laser (Green)" 3
+		"Sidereal VLS Array"
+		"Equinox Torpedo Stockpile" 3
+		"Equinox Torpedo" 36
+		"Sidereal Missile" 126
+		
+		"Radiant Fusion Core"
+		"Ultracapacitor Bank" 2
+		"Specular Deflector"
+		"Optical Lasing Generator" 2
+		"Electrostatic Ramscoop" 2
+		"Multispectral Scanner" 9
+		"Electromotive Rifle" 90
+		
+		"Z-600 Fusion Torch"
+		"XY-420 Radial Thrusters"
+		"Hyperdrive"
+
+	engine 0 250
+	"reverse engine" -9 234
+		angle -30
+	"reverse engine" 9 234
+		angle 30
+	"steering engine" -12 -258
+		angle 90
+		right
+	"steering engine" 12 -258
+		angle -90
+		left
+	"steering engine" -9 234
+		angle 90
+		left
+	"steering engine" 9 234
+		angle -90
+		right
+
+	gun -18 -126
+		angle -90
+	gun 18 -126
+		angle 90
+	gun -15 -108
+		angle -90
+	gun 15 -108
+		angle 90
+	turret 0 -252 "Phased Array Laser (Green)"
+	turret 60 -174 "Phased Array Laser (Blue)"
+	turret -60 -174 "Phased Array Laser (Blue)"
+	turret 0 -96
+	turret 0 -36 "Sidereal VLS Array"
+	turret 0 30 "Phased Array Laser (Green)"
+	turret 0 96 "Phased Array Laser (Green)"
+	turret 0 160
+
+	bay "Drone" -24 -234
+	bay "Drone" 24 -234
+	bay "Drone" -42 -225
+	bay "Drone" 42 -225
+	bay "Drone" -63 -219
+	bay "Drone" 63 -219
+	bay "Drone" -18 -216
+	bay "Drone" 18 -216
+	bay "Drone" -24 -175
+	bay "Drone" 24 -175
+	bay "Drone" -24 -175
+	bay "Drone" 24 -175
+	bay "Drone" -46 -165
+	bay "Drone" 46 -165
+	bay "Drone" -46 -165
+	bay "Drone" 46 -165
+	bay "Drone" -18 -150
+	bay "Drone" 18 -150
+	bay "Fighter" 0 -216
+		over
+	bay "Fighter" 0 -216
+		under
+	bay "Fighter" 0 -150
+		over
+	bay "Fighter" 0 -150
+		under
+	bay "Fighter" 0 -117
+		over
+	bay "Fighter" 0 -117
+		under
+
+	leak "leak" 30 60
+	leak "flame" 30 90
+	leak "big leak" 30 30
+	explode "tiny explosion" 36
+	explode "small explosion" 42
+	explode "medium explosion" 45
+	explode "large explosion" 36
+	explode "huge explosion" 6
+	"final explode" "final explosion large"
+
+	description "The Atenuatia is an Avgi fleet carrier, a capital ship that makes up the centerpiece of their battlefleets. With six wings of combat drones and two wings of bombers, an Atenuatia can bring forth an astounding amount of firepower. Carrying this arsenal of fighters has its cost, however, and the Atenuatia can carry relayively few heavy weapons for its size."
+
+
+# Fighters
+
+ship "Akoustiki"
+	sprite "ship/avgi akoustiki"
+	thumbnail "thumbnail/avgi akoustiki"
+	attributes
+		category "Fighter"
+		licenses
+			"Avgi Civilian"
```

Comment:
```suggestion
		licenses
			"Avgi Civilian"
		"cost" 148000
```
Notably higher than the Dropship's 91,000, but lower than the Boxwing's 369,000.